src/Microsoft.AspNetCore.Razor.Language/RazorExtensionInitializer.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public abstract class RazorExtensionInitializer
+    {
+        public abstract void Initialize(RazorProjectEngineBuilder builder);
```

Comment:
Should we add an empty ctor to this since we expect it to be initializable without args?

Reply:
I thought about that but it doesn't really do anything. Anyone who subclasses can just define another constructor. I can't think of a way to force a derived class to do the right thing.